### ISSUE TITLE
There is no way to save the contents of the writer session

### ISSUE THREAD

**Issue by raphydaphy** [raphydaphy/Q-OS#9]
Although with changes in Update 2.x it should technically be possible, when you close the <code>writer</code> session, you loose anything you wrote in that session and you must rewrite it from scratch when you next open the <code>writer</code> interface.

**Comment by raphydaphy** [comment]
I'm going to try to use the vfs to make this work

**Comment by plankp** [comment]
I think we should try implementing `FAT32` or something like `ext2` because there is actually documentation on those storages

**Comment by raphydaphy** [comment]
Yea I have been reading about it on the OsDev wiki for the last few days and also on some other places because filesystems obviously are kindof hard to implement and I want to use FAT32 :) Tomorrow is the weekend so I can probably finish it then if not today :+1:

**Comment by raphydaphy** [comment]
Maybe VFAT though because it lets you have longgg file names

**Comment by plankp** [comment]
k

**Comment by plankp** [comment]
I've got to homework

**Comment by raphydaphy** [comment]
:+1: same

**Comment by plankp** [comment]
(grade 10)

**Comment by raphydaphy** [comment]
<img width="494" alt="screen shot 2015-11-20 at 11 33 57 am" src="https://cloud.githubusercontent.com/assets/8121164/11288727/9d5f8bfe-8f7a-11e5-9699-1fda467c09f0.png">

grade 7 :cry:

**Comment by raphydaphy** [comment]
<img width="562" alt="screen shot 2015-11-19 at 8 13 52 pm" src="https://cloud.githubusercontent.com/assets/8121164/11288743/c02b8c3c-8f7a-11e5-9f6e-deb9b42031c2.png">

**Comment by plankp** [comment]
(errors)

**Comment by raphydaphy** [comment]
:disappointed:

**Comment by plankp** [comment]
>File table. Don't think of this as just a simple table containing a list of files and their locations. One idea I've had and never implemented is, **instead of storing files, the system would store file parts, and the file table would list the parts in each file. This would be useful for saving space if many files on the disk are the same or similar (for example, license agreements).**

-- http://wiki.osdev.org/File_Systems

**Comment by raphydaphy** [comment]
I'm working mostly on Q PL but I am trying to do this also....
but i unassigned myself anyway

**Comment by telip007** [comment]
Is it now possible to save the writer session ?

**Comment by raphydaphy** [comment]
No,
I unassigned myself because I will be mostly working on [Q PL](github.com/raphydaphy/Q-Programming-Language) for the next few days, so if someone else wanted to do this then they can :+1:

**Comment by telip007** [comment]
Ok I will look into it ;)

**Comment by raphydaphy** [comment]
:+1:

**Comment by telip007** [comment]
If I'm allowed to 😁😁

**Comment by raphydaphy** [comment]
Y would you not be.....

**Comment by telip007** [comment]
little joke :)

**Comment by raphydaphy** [comment]
:+1: :white_check_mark:

**Comment by telip007** [comment]
I didn't have any time to look at it. Is there a filesystem implemented by you or do we still need this ?

**Comment by telip007** [comment]
found it in the wiki ;)

**Comment by plankp** [comment]
which wiki?

**Comment by plankp** [comment]
(love to help but don't know where to start)

**Comment by telip007** [comment]
https://github.com/raphydaphy/Q-Operating-System/wiki 
this wiki ? :D

**Comment by plankp** [comment]
Its a VFS though

**Comment by plankp** [comment]
Has nothing to do with the hard drive

**Comment by telip007** [comment]
Yeah but my question was is there a file system not, is there a file system which has something to do with the hard drive 😬😬

**Comment by plankp** [comment]
ok?

**Comment by telip007** [comment]
If there were no kind of a file system, I would create new files for the file system implementation, but now i only have to change it ;)

**Comment by plankp** [comment]
ok

**Comment by telip007** [comment]
I hope I could explain it :D

**Comment by telip007** [comment]
http://wiki.osdev.org/User:Requimrar/FAT32
If someone else wants to look at this, here is a nice article, the code is written in c++ but it gives you a better look at FAT32 :)

**Comment by plankp** [comment]
ok

**Comment by Pvanduyse** [comment]
I think we need to make this into a milestone.

**Comment by Pvanduyse** [comment]
Can someone take all the steps required for this, and break them down into separate smaller items all under the `Writer Saving` milestone?

**Comment by krisnarocks** [comment]
We need to make file system first, right?
